### PR TITLE
Update java's classpaths 

### PR DESCRIPTION
**Pull request by avlitman**
Signed-off-by: Aviv Litman <alitman@redhat.com>

### CODE REVIEW

File: packaging/bin/dwh-classpath.sh [hunk 1]
```diff
@@ -47,20 +47,23 @@ if [ -x /usr/bin/java-config ]; then
 	done
 elif [ -x /usr/bin/build-classpath ]; then
 	dom4j="$(build-classpath dom4j 2> /dev/null)"
-	[ -z "${dom4j}" ] && dom4j="$(build-classpath dom4j-eap6 2> /dev/null)"
```
Comment (arso) [inline packaging/bin/dwh-classpath.sh:50]: Was it intended to be removed?

**Review by mwperina — APPROVED** [review]
+1

**Pull request merged**